src/mine-sweeper.tsx

```diff
@@ -176,16 +180,40 @@ export default class MineSweeper extends React.Component<IMineSweeperProps, IMin
     }
   }
 
-  private revealCell(x: number, y: number) {
-    const mineField = this.state.mineField;
-    mineField[x][y].isRevealed = true;
-    mineField[x][y].isFlagged = false;
-    this.setState({
```

Comment:
why you removed this? Thats used to set some classes based on those flags which are set in line 181-182

Reply:
that was replaced not removed i think, replaced for more modular that can be used on the reveal surrounding functions

Reply:
I think theres no point in moving this down into the `revealSurroundingCells` since the first thing we want to do is reveal the cell for `x,y`. No need to abstract away the `setState` function either, we don't gain anything from that.